src/lightning/pytorch/loops/dataloader/evaluation_loop.py

```diff
@@ -26,6 +26,7 @@
 from lightning.pytorch.loops.epoch import _EvaluationEpochLoop
 from lightning.pytorch.loops.fetchers import _DataFetcher
 from lightning.pytorch.loops.utilities import _select_data_fetcher, _set_sampler_epoch
+from lightning.pytorch.trainer import call
```

Comment:
Then also rather import each function then this module call which may confuse as being another object...

Reply:
Doing it like this follows the pattern started by William at https://github.com/Lightning-AI/lightning/pull/14871 and https://github.com/Lightning-AI/lightning/pull/14862